packages/zowe-explorer-api/src/vscode/ui/PaginationCodeLens.ts

```diff
@@ -12,7 +12,7 @@
 import * as vscode from "vscode";
 
 export class PaginationCodeLens implements vscode.CodeLensProvider {
-    public constructor(private commandId: string) {}
+    public constructor(private commandId: string, private shouldShowCodeLens?: Function) {}
```

Comment:
Can we make the type for `shouldShowCodeLens` a bit more restrictive? This ensures that the caller provides a function that takes in a single parameter (`document`) and returns a `boolean` value:

```ts
public constructor(private commandId: string, private shouldShowCodeLens?: (document: vscode.TextDocument) => boolean) {}
```